app/controllers/users/passwords_controller.rb

```diff
@@ -27,26 +28,10 @@ def user_params
     end
 
     def handle_success
-      re_encrypt_active_profile
-
       bypass_sign_in current_user
 
+      flash[:recovery_code] = @update_user_password.recovery_code
```

Comment:
are we putting the recovery code itself in the flash? if not, wondering if there is a clearer name we could pick